src/en/about-juju.md

```diff
@@ -41,11 +40,12 @@ in seconds, on any cloud or server or virtual machine.
 ## Can I use Juju with Puppet or Chef or Ansible?
 
 Yes! Puppet, Chef, and Ansible are great tools for writing configuration
-files. Juju works a layer above that by focusing on the service the
-application delivers, regardless of the machine on which it runs. So the Juju
-charm for an application includes (amongst other things) all the logic for
-writing configuration files for that application - that logic itself can be
-written in whatever language or tool the author of the charm prefers.
+files. Juju works a layer above that by focusing on the long term operations
+required to keep that software running over time, regardless of the machine on
+which it runs. So the Juju charm for an application includes (amongst other
+things) all the logic for writing configuration files for that
+application - that logic itself can be written in whatever language or tool
+the author of the charm prefers.
```

Comment:
Same here, I think that 

    long term operations

vs

    the service the-application delivers

is more in line with our current messaging.